experimental/PropertyDDS/packages/property-common/src/index.ts

```diff
@@ -3,14 +3,32 @@
  * Licensed under the MIT License.
  */
 
-export * from "./constants";
-export * from "./consoleUtils";
-export * from "./chronometer";
-export * from "./joinPaths";
-export * from "./guidUtils";
-export * from "./error_objects";
-export * from "./deferredPromise";
-export * from "./deterministicRandomGenerator";
-export * from "./hashCalculator";
-export * from "./datastructures";
+export { Chronometer } from "./chronometer";
+export { ConsoleUtils } from "./consoleUtils";
+export { constants } from "./constants";
+export {
+	BaseDataArray,
+	BoolDataArray,
+	Collection,
+	Float32DataArray,
+	Float64DataArray,
+	Int16DataArray,
+	Int32DataArray,
+	Int64,
+	Int8DataArray,
+	Integer64,
+	SortedCollection,
+	StringDataArray,
+	Uint16DataArray,
+	Uint32DataArray,
+	Uint64,
+	Uint8DataArray,
+	UniversalDataArray,
+} from "./datastructures";
+export { DeferredPromise } from "./deferredPromise";
+export { DeterministicRandomGenerator } from "./deterministicRandomGenerator";
 export { EventEmitter } from "events";
```

Comment:
Very strange that this is being reexported in the first place, but probably should be pulled up above the exports from local files.